lib/v1/index.js

```diff
@@ -1,14 +1,13 @@
 const { Datatypes, errors, extractDomain } = require('./utils');
 const Config = require('./config');
 const { Crypto, Http, Forms } = require('./core');
-const extractDomainFromUrl = require('./utils/domain');
 
 class EvervaultClient {
-  constructor(teamId) {
+  constructor(teamId, configUrls) {
     if (!Datatypes.isString(teamId)) {
       throw new errors.InitializationError('teamId must be a string');
     }
-    this.config = Config(teamId);
+    this.config = Config(teamId, configUrls);
```

Comment:
Only hand off the urls that we're setting to the config for now. (Will need some type checks to make sure `config` is defined and fallback to an empty object if not
```suggestion
    this.config = Config(teamId, config.urls);
```